### PR TITLE
Add new printlns

### PR DESCRIPTION
**Pull request by a-r-t**
Added some more printlns to application to make it better

### CODE REVIEW

**Comment by a-r-t** [comment]
This guy stinks!!!!

**Comment by BrianWanamaker** [comment]
Spaghetti code

File: HelloWorld.java [hunk 1]
```diff
@@ -3,6 +3,8 @@ public class HelloWorld {
 
     public static void main(String[] args) {
         System.out.println("Hello world!");
+        System.out.println("Hello branchies!!!");
```
Comment (a-r-t) [inline HelloWorld.java:6]: That is the worst word I've ever heard...

**Comment by a-r-t** [comment]
> Spaghetti code

:( that is not appropriate in the gryffindor brotherhood

**Pull request merged**